Can't compile in no std environment

I tried using the graphlib on a embedded project I'm working on, but failed to compile it due to dependencies to the std crate. 

For reproducibility, I set up the environment the same as described in the embedded rust book: https://docs.rust-embedded.org/book/start/qemu.html

The std features should be disabled:
```toml
[dependencies]
graphlib = {version = "0.6.1", features = ["no_std"]}
```

When compiled with: 
```bash
cargo +nightly build --target thumbv7m-none-eabi
```
I got the following error:
```bash
   Compiling lazy_static v1.4.0
   Compiling hex v0.4.2
   Compiling num_cpus v1.12.0
   Compiling ahash v0.2.18
error[E0463]: can't find crate for `std`
  |
  = note: the `thumbv7m-none-eabi` target may not be installed

error: aborting due to previous error

For more information about this error, try `rustc --explain E0463`.
error[E0463]: can't find crate for `std`
  |
  = note: the `thumbv7m-none-eabi` target may not be installed

error: aborting due to previous error

For more information about this error, try `rustc --explain E0463`.
error: could not compile `hex`.

To learn more, run the command again with --verbose.
warning: build failed, waiting for other jobs to finish...
error: could not compile `num_cpus`.

To learn more, run the command again with --verbose.
warning: build failed, waiting for other jobs to finish...
error[E0463]: can't find crate for `std`
 --> /Users/schuepbs/.cargo/registry/src/github.com-1ecc6299db9ec823/lazy_static-1.4.0/src/inline_lazy.rs:9:1
  |
9 | extern crate std;
  | ^^^^^^^^^^^^^^^^^ can't find crate
  |
  = note: the `thumbv7m-none-eabi` target may not be installed

error: aborting due to previous error

For more information about this error, try `rustc --explain E0463`.
error: could not compile `lazy_static`.

To learn more, run the command again with --verbose.
warning: build failed, waiting for other jobs to finish...
error: build failed
```

I believe that this is due to a dependency in the hashbrow crate.
Do you know anything about this?

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -14,24 +14,21 @@ readme = "README.md"
 travis-ci = { repository = "purpleprotocol/graphlib", branch = "master" }
 
 [dependencies]
-rand = { version = "0.7.2", default-features = false }
-rand_core = "0.5.1"
-rand_isaac = "0.2.0"
-hex = "0.4.0"
-hashbrown = { version = "0.6.3", features = ["rayon"] }
-dot = {version = "0.1.4", optional=true}
+rand = { version = "0.7.3", default-features = false }
+rand_core = { version = "0.5.1", default-features = false } 
+rand_isaac = { version = "0.2.0", default-features = false }
+hex = { version = "0.4.0", default-features = false }
+hashbrown = { version = "0.6.3", default-features = false, features = ["inline-more", "ahash"] }
+dot = { version = "0.1.4", optional = true }
 
 [dev-dependencies]
 criterion = "0.3.0"
-# Due to criterion dependency on std
-rand = "0.7.2"
 
 [[bench]]
 name = "benchmark"
 harness = false
 
 [features]
-
 default = []
 # use `cargo bench --features sbench` only if you want benchmarks with 10 million
 # iterations (may fail on some systems)

